pkg/app/pagewriter/static_pages_test.go

```diff
@@ -0,0 +1,146 @@
+package pagewriter
+
+import (
+	"errors"
+	"html/template"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Static Pages", func() {
+	var customDir string
+	const customRobots = "I AM A ROBOT!!!"
```

Comment:
NIT: Can we have this custom test payload resemble the robots.txt traditional structure?